analyze_openapi_changes.py

```
@@ -0,0 +1,114 @@
+        summary_lines.append(f"- **Security changes:** {api_changes['security_changes'] or 'None'}\n")
+    if has_prod_changes:
+        summary_lines.append("## Production Changes")
+        summary_lines.append(f"- **Documentation updates:** {production_changes['documentation_updates'] or 'None'}")
+        summary_lines.append(f"- **Example updates:** {production_changes['example_updates'] or 'None'}")
+        summary_lines.append(f"- **Internal changes:** {production_changes['internal_changes'] or 'None'}\n")
+
+    return classification, "\n".join(summary_lines)
+
+def save_markdown(summary_md: str, output_dir: str, repo_name: str) -> None:
+    os.makedirs(output_dir, exist_ok=True)
+    current_date = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)